mvp_cd/deploy_app/models.py

```diff
@@ -3,21 +3,24 @@
 # Create your models here.
 
 
-class StagingBuildInfo(models.Model):
-    curr_build_info = models.TextField()
-    is_success = models.BooleanField(default=False)
-
+class ServerType(object):
+    APP_SERVER = 'app_server'
+    WORKER = 'celery_worker'
 
-class StagingWorkerBuildInfo(models.Model):
-    curr_build_info = models.TextField()
-    is_success = models.BooleanField(default=False)
 
-
-class ProductionBuildInfo(models.Model):
-    curr_build_info = models.TextField()
-    is_success = models.BooleanField(default=False)
+class BuildInfo(models.Model):
+    circleci_json = models.TextField()
+    branch = models.CharField(max_length=255)
+    commitor = models.CharField(max_length=255)
+    circleci_url = models.CharField(max_length=255)
+    created_at = models.DateTimeField(auto_now_add=True)
+    vcs_revision = models.TextField(default='0000000')
```

Comment:
TextField is used for storing large data. Won't `varchar` be sufficient here ? Also default doesn't make sense.